websites/R/Rabbit's BS Viewer/presence.ts

```diff
@@ -0,0 +1,69 @@
+﻿const presence = new Presence({
+  clientId: "839924185278840853"
+  }),
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const elapsed = await presence.getSetting("elapsed"),
+    timeleft = await presence.getSetting("timeleft"),
+    buttons = await presence.getSetting("buttons"),
+    presenceData: PresenceData = {
+      largeImageKey: "logo",
+      startTimestamp: browsingStamp
+    };
+
+  if (document.location.pathname.includes("/bs-viewer/")) {
+    if (document.querySelector("#songname").textContent == "") {
```

Comment:
```suggestion
    if (document.querySelector("#songname").textContent === "") {
```
we always check type 😉